README.md

```diff
@@ -1,21 +1,89 @@
-# React from Markup
+# react-from-markup
+> Convert static markup into React components, so you can control React props in a static environment.
```

Comment:
I don't think I like this. Would probably prefer to stick with original.

Reply:
Hm, I'm not sure I would understand what that meant. What about mentioning `data-` attrs.

> Control your React props with `data-` attributes

Or similar to  your original

> Declare your React props with `data-` attributes